dashboard/app/models/ability.rb

```diff
@@ -221,7 +222,7 @@ def initialize(user)
       ]
```

Comment:
Note: Levelbuilders regain the ability to index Level in the block above, when they're given permission to manage Level.